### PR TITLE
Pks proxy

### PR DESCRIPTION
**Pull request by lparis**
for 1.3, out for review

### CODE REVIEW

**Comment by bentarnoff** [comment]
Thanks @lparis. Should I block it on VMW review?

**Comment by lparis** [comment]
Yes please. Thanks.

File: proxies.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Select Proxy Settings.
+
+1. Under HTTP Proxy, enter the FQDN or IP address of the HTTP proxy endpoint. For example, `http://proxy.pks.com:1234`.
+
+1. Under HTTPS Proxy, enter the FQDN or IP address of the HTTPS proxy endpoint. For example, `https://proxy.pks.com:1234`.
+
+1. Under No Proxy, **include** the following networks that must bypass the proxy (required).
+
+		`<Deployment-Network-Cidr>,<Node-Network-IP-Block-Cidr>,<Pod-Network-IP-Block-Cidr>`
```
Comment (shyaamsn) [inline proxies.html.md.erb]: No_proxy for Ops manager:
'127.0.0.1,localhost,<deployment-network-cidr>'

File: proxies.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## <a id='pks-proxy'></a>PKS Proxy
+
+In a PKS environment on vSphere with NSX-T, PKS components are typically deployed to a management network/subnet separate from the standard network with vCenter, NSX-T Manager, and ESXi hosts. PKS Proxy is for such environments where there is an HTTP/S proxy between the PKS network and the main network.
+
+PKS Proxy settings are applied to the PKS API instance. When a PKS operator creates a Kubernetes cluster, the PKS API instance VM behind a proxy is able to manage objects on the NSX-T Manager.
+
+    <%= image_tag("pks-proxy-config.png") %>
+
+To configure a proxy for PKS:
```
Comment (shyaamsn) [inline proxies.html.md.erb]: To configure a proxy for Bosh/Ops manager

File: proxies.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+1. Under No Proxy, **include** the following networks that must bypass the proxy (required).
+
+		`<Deployment-Network-Cidr>,<Node-Network-IP-Block-Cidr>,<Pod-Network-IP-Block-Cidr>`
+1. Save.
+
+1. Review pending changes in Ops Manager.
+
+1. Deploy PKS components using Ops Manager.
+
+## Kubernetes Cluster Proxy
```
Comment (shyaamsn) [inline proxies.html.md.erb]: Kubernetes cluster and PKS proxy

**Comment by lparis** [comment]
Reviewed in person with Dev. Changes mage. Ready for review and merge. Not sure if this applies to 1.2.3 or 1.3.0.

**Comment by lparis** [comment]
This feature is for 1.3, so it only applies to Master. thanks

File: proxies.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+1. Select **Proxy Settings**.
+
+1. Under **HTTP Proxy**, enter the FQDN or IP address of the HTTP proxy endpoint. For example, `http://myproxy.com:1234`.
+
+1. Under **HTTPS Proxy**, enter the FQDN or IP address of the HTTPS proxy endpoint. For example, `https://myproxy.com:1234`.
+
+1. Under **No Proxy**, include the following networks that must bypass the proxy (required).
+
+		`127.0.0.1,localhost,<Deployment-Network-CIDR>`
```
Comment (shyaamsn) [inline proxies.html.md.erb]: Tano found that CIDR's dont work here. So, we need to add individual IP's.

`127.0.0.1,localhost,<bosh-director-ip>,<pks-vm-ip>`

Bosh director ip is the first IP in the deployment network CIDR
PKS VM ip is the second IP in the deployment network CIDR

Comment (lparis) [reply proxies.html.md.erb]: oh, good catch, will update, thanks

**Comment by bentarnoff** [comment]
@lparis Please let us know when this is ready for merge

**Comment by bentarnoff** [comment]
@lparis I've done a light edit. Please take a look: https://docs-pcf-staging.cfapps.io/runtimes/pks/1-3/proxies.html

**Comment by lparis** [comment]
Will do. Thanks. Also will have the devs review it as well. Thanks.

**Pull request merged**